Source/WebCore/editing/TextIterator.cpp

```diff
@@ -642,19 +642,22 @@ void TextIterator::handleTextRun()
         auto nextTextRun = InlineIterator::nextTextBoxInLogicalOrder(m_textRun, m_textRunLogicalOrderCache);
 
         if (runStart < runEnd) {
-            auto isNewlineOrTab = [&](UChar character) {
-                return character == '\n' || character == '\t';
-            };
+            Function<bool(UChar)> isCharacterThatShouldBeReplaceBySpace;
```

Comment:
`ScopedLambda<bool(Char)>` can avoid heap allocation if this function is scoped.
Or since isCharacterThatShouldBeReplaceBySpace function is not referencing to any captured variables, we can use function pointer here.

Reply:
This code is hot so please avoid heap allocation.